src/renderers/shared/ReactDebugTool.js

```diff
@@ -68,7 +69,7 @@ var ReactDebugTool = ((null: any): typeof ReactDebugTool);
 if (__DEV__) {
   const hooks = [];
   const didHookThrowForEvent = {};
-  const ReactComponentTreeHook = require('react/lib/ReactComponentTreeHook');
+  const ReactComponentTreeHook: ComponentTreeHookDevType = (require('react/lib/ReactComponentTreeHook'): any);
```

Comment:
Yes this fails: `Uncaught TypeError: Cannot read property 'purgeUnmountedComponents' of undefined`